tests/integration/test_wpk/test_wpk_manager.py

```diff
@@ -15,19 +15,19 @@
 import platform
```

Comment:
flake8 reports a bunch of style errors on this script, including some unused imports, and unused local variables.

Reply:
Removed in 29484935443c9d15880a267f5abcd72f8779aa3a